src/components/Vault/VaultRow.tsx

```diff
@@ -42,12 +45,29 @@ const VaultRow: FC<VaultProps> = (props) => {
       { name: "offset", options: { offset: [-3, 4] } },
     ],
   })
+  const [updatedProps, setUpdatedProps] = useState<VaultProps>(props)
+  const isToken = useIsTokenCompounder(props.type)
 
-  const { isLoading: isLoadingAsset } = useCompounderPoolAsset(props)
-  const { isLoading: isLoadingUnderlying } =
-    useCompounderUnderlyingAssets(props)
+  const { data: ybTokenAddress } = useCompounderYbTokens({
+    address: props.address,
+    type: props.type,
+  })
+
+  useEffect(() => {
```

Comment:
Render will trigger every time props change and every time ybTokenAddress changes. 

If you want to pass `ybTokenAddress` instead of props.address to the child components, you should just do that:

```tsx
const childProps = {
  address: ybTokenAddress ?? "0x",
  type: props.type
}

// ...
return (
  <>
    // ...
    <VaultDepositedLp {...childProps} />
  </>
)

```

Reply:
done